universal-login-relayer/lib/relayer.js

```diff
@@ -21,15 +22,31 @@ function errorHandler (err, req, res, next) {
 }
 
 class Relayer {
-  constructor(config, provider = '') {
+  constructor(config, provider = '', database) {
     this.port = config.port || defaultPort;
     this.config = config;
     this.hooks = new EventEmitter();
     this.provider = provider || new providers.JsonRpcProvider(config.jsonRpcUrl, config.chainSpec);
     this.wallet = new Wallet(config.privateKey, this.provider);
+    this.database = database;
   }
 
-  start() {
+  async start() {
+    if (await this.checkIfAllMigrated()) {
+      this.runServer();
+    } else {
+      this.database.destroy();
+      throw Error('Database is out of date.');
```

Comment:
Make more descriptive, e.g. sth like:`You need to run migrations. Type '$ knex migrate:latest'`

Reply:
✅